mflowgen/components/graph.py

```diff
@@ -127,6 +136,53 @@ def get_edges_o( s, step_name ):
       return s.sort_edges(s._edges_o[ step_name ])
     except KeyError:
       return []
+  
+  def add_input( s, name, *args ):
```

Comment:
What happens if you add_input on an input handle, which is already connected to some other node's output?

Does the official graph add_input "win"?

Reply:
Yes, when the graph is used as a subgraph, the graph-level input takes priority.